src/crud.js

```diff
@@ -23,5 +23,7 @@ export function removeTodo(event) {
 
 export function clearCompleted() {
```

Comment:
Mean reason for writing a unit test is to figure out when we have a bug in the actual function itself.

More explanation:
- You have a function called `clearCompleted ` in `crud.js`
- You wrote a test for this function in `crud.test.js`
- Introduce a bug in clearCompleted` function in `crud.js` I'm expecting the test to fail but it was passed.

In your case;
So try to introduce a bug in this function by changing `!todo.completed` to `todo.completed`, then run the test.

The test suite still passed, the test suite should fail.

Hint: 
You're mocking the functions, you're supposed to mock only `localStorage updates` and `HTML elements to test DOM manipulation functions`